src/redux/actionTypes/differentialExpression.js

```diff
@@ -16,6 +16,19 @@ const DIFF_EXPR_LOADED = `${DIFF_EXPR}/loaded`;
 
 const DIFF_EXPR_ERROR = `${DIFF_EXPR}/error`;
 
+/**
+ * Sets the state for diffential expression settings
+ *
+ */
+const DIFF_EXPR_COMPARISON_TYPE_SET = `${DIFF_EXPR}/comparisonType`;
+const DIFF_EXPR_COMPARISON_GROUP_SET = `${DIFF_EXPR}/comparisonGroup`;
```

Comment:
Any particular reason why we can't merge them and have one action about this?

Reply:
`comparisonType` fires when user changes the active tab. That does not mean there are settings changed, which will be fire `comparisonGroup`. So I think these 2 actions have to be separate.

Reply:
By "the active tab" do you mean when the user changes the tab under `Tools` from `Gene List` to `Differential Expression`?

Reply:
No, I mean switching between the two radio buttons to switch the type of differential expression